dts/arm/nordic/nrf51822.dtsi

```diff
@@ -20,6 +20,8 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 
+			label="NRF5_FLASH";
```

Comment:
Seems like this should be "NRF5_FLASH_CTRL";

So we can also have a label on the flash itself if needed.

Reply:
NRF5_FLASH is used to identify the nordic flash driver (along with device_get_bindings call). I would change this to something like NRF5_FLASH_NAME. ?